### ISSUE TITLE
Hard Crash: FATAL:hwnd_util.cc(81)] 87

### ISSUE THREAD

**Issue by uptodatehuang** [cefsharp/CefSharp#3411]
**What version of the product are you using?**
NuGet: 
cef.redist.x64(87.1.13), 
cef.redist.x86(87.1.13)
CefSharp.Common(87.1.132)
CefSharp.WinForms(87.1.132)
VCRuntime.CefSharp.140(1.0.4)

**What architecture x86 or x64?**
x86

**On what operating system?**
Win10

**Are you using WinForms, WPF or OffScreen?**
WinForms

**What steps will reproduce the problem?**
The program opens a winform panel with a ChromiumWebBrowser.
The panel window size is automatically adjusted base on user predefined size

**What is the expected output? What do you see instead?**
Expected: The Cef is initialized and see the google page.
Actual: the Cef is not initialized and the application is crashed.

**Please provide any additional information below.**
Maybe it worth mention that,  I have tested our app with CefSharpV86, it works fine. the issue seems to appear only from V87 onwards.

**Application Event Viewer has this:**
Faulting application name: DEVApp.exe, version: 4.5.2.24056, time stamp: 0x5f87b243
Faulting module name: libcef.dll, version: 87.1.13.0, time stamp: 0x5ff502f4
Exception code: 0x4000001f
Fault offset: 0x020a34ed
Faulting process id: 0xfa8
Faulting application start time: 0x01d70972f28f06ff
Faulting application path: C:\Apps\DEVApp.exe
Faulting module path: C:\Users\..\AppData\Local\DEVApp\libcef.dll
Report Id: 45bb29e3-6e4f-4e22-9cc2-d729d3785d73
Faulting package full name: 
Faulting package-relative application ID: 

**The file cef.log from the bin folder has this:**
(ps: cef log lack of detail for this issue)
[0223/103235.232:FATAL:hwnd_util.cc(81)] 87

**- Any other background information that's relevant? Are you doing something out of the ordinary? 3rd party controls?**

**Does this problem also occur in the CEF Sample Application from http://opensource.spotify.com/cefbuilds/index.html?**
I have tested running this example app locally, it works fine. https://github.com/cefsharp/CefSharp.MinimalExample/tree/cefsharp/87/CefSharp.MinimalExample.WinForms

**Comment by amaitland** [comment]
Closing as duplicate of #2378

As stated in https://github.com/cefsharp/CefSharp/issues/2378#issuecomment-780194746 and actual example that reproduces the problem is required to reopen the issue.

**Comment by amaitland** [comment]
> I have tested running this example app locally, it works fine. https://github.com/cefsharp/CefSharp.MinimalExample/tree/cefsharp/87/CefSharp.MinimalExample.WinForms

If you'd like to provide an example then please fork the `CefSharp.MinimalExample`, squash your changes into a single commit then push them to `GitHub`, thanks.

I'm happy to reopen #2378 I just need an actual example, your `What steps will reproduce the problem?` don't provide a meaningful reproduction of the issue. From my point of view it's just a vague description.

**Comment by uptodatehuang** [comment]
Hi, I have found a fix to our issue.

the window form app we have is a multi-tabs application, mix of c# pages and cefsharp pages. 
on user login, it will dynamically initiating tabs base on user profile. 

the issue we found is, we are lazily initializing Cef during tabs setup. This seems to works fine when we are with old Cef version 63, but no longer when we are starting to migrate to Cef version 87.

the fix we did is, Eagerly initializing Cef on application start up, before tabs setup. 

------------

BTW, not sure here is the best place to ask, @amaitland do you have a timeline for v88 release? we are actually waiting on this release version, since it includes a security fix for chrome. https://www.theregister.com/2021/02/05/chrome_zero_day_update/

**Comment by amaitland** [comment]
> do you have a timeline for v88 release?

https://github.com/cefsharp/CefSharp/releases/tag/v88.2.40-pre is already available which is based on `Chromium 88.0.4324.150`. The final release is planned for this week. You can subscribe to release only notifications here on `GitHub` to be notified when there is a release.

If you are using `CefSharp` in a commercial application please consider [sponsoring](https://github.com/sponsors/amaitland) it's development.
